src/common/transformations/src/transformations/common_optimizations/optimize_strided_slice.cpp

```diff
@@ -258,7 +258,7 @@ bool ov::pass::GroupedStridedSliceOptimizer::run_on_model(const std::shared_ptr<
             }
             ++i;
         }
-        copy_runtime_info(ops_to_replace, variadic_split);
+        copy_runtime_info(ops_to_replace, {variadic_split, axis_const, size_splits_const});
```

Comment:
do we need to update rt_info checks in the tests to catch this?

Reply:
> do we need to update rt_info checks in the tests to catch this?

Actually it is hard to catch. There is a workaround in copy_runtime_info which adds Constant to targets if they have no rt_info yet https://github.com/openvinotoolkit/openvino/blob/9bedafb560fd4bc0e4627c8cac67cc0236a72e86/src/core/src/rt_info.cpp#L85 But I think it is better to add them explicitly.